cookbook/blending-with-context/README.md

```diff
@@ -0,0 +1,41 @@
+# Blending with Context
+
+There are situations when you need to blend the same token with multiple URIs but you want only one of them to be invoked depending on the context. For example a network support person should have access to an employee record but it needs to be scoped to only relative information with no edit rights while a system admin can have a full editable view of employee. Now there are two different roles trying to access the same entity but in different ways, this is where Context comes into play. 
+
+Contexts is an array of string that helps filter out the blending one step further. Two blending matches if both have the same token and context. In the previous example you can blend Employee with two different contexts:
```

Comment:
array of string -> array of strings

This sentence is confusing: "Two blending matches if both have the same token and context." The problem is that you're using the word blending to describe a blending point and the concept of visually integrating two applications.

Also, "both" is redundant. I would rephrase to:
> Two blending points (?) match if they have the same token and context

Reply:
And Employee should be `Employee` since it's a class

Reply:
I would also run the text through something https://app.grammarly.com/ to fix some comma placements.